### PR TITLE
add info on how to paginate the same model multiple times

### PR DESCRIPTION
**Pull request by swiffer**

### CODE REVIEW

File: en/controllers/components/pagination.rst [hunk 1]
```diff
@@ -208,6 +208,48 @@ elements and URLs for pagination.
...
+                'id' => 'desc',
+            ],
+        ],
+    ];
+    
+    TableRegistry::config('UnpublishedArticles', [
+        'className' => 'App\Model\Table\ArticlesTable',
+        'table' => 'articles',
+        'entityClass' => 'App\Model\Entity\Article',
+    ]);
```
Comment (markstory) [inline en/controllers/components/pagination.rst]: Should this be in the controller? Or should we tell the reader to place it in the app bootstrap or in the relevant model file?

Comment (swiffer) [reply en/controllers/components/pagination.rst]: Good question. As I needed this for a single controller action/view I preferred placing it within this action so I clearly know what it was used for.

Comment (markstory) [reply en/controllers/components/pagination.rst]: ok.

Comment (markstory) [reply en/controllers/components/pagination.rst]: Should there be a comment above this call explaining what it does? I don't think `TableRegistry::config()` is a frequently used method.

Comment (swiffer) [reply en/controllers/components/pagination.rst]: Has been solved by the latest commits already

**Comment by swiffer** [comment]
@dereuromark anything still missing? Solved the discussion

File: en/controllers/components/pagination.rst [hunk 2]
```diff
@@ -208,6 +208,49 @@ elements and URLs for pagination.
...
+
+    $published_articles = $this->paginate(
```
Comment (dereuromark) [inline en/controllers/components/pagination.rst]: cs: camelBacked, not underscored

File: en/controllers/components/pagination.rst (hunk 1 above)
Comment (dereuromark) [reply en/controllers/components/pagination.rst]: "Should there be a comment above this call explaining what it does?"
Dont see that added yet.

Comment (swiffer) [reply en/controllers/components/pagination.rst]: :) should have read more carefully

File: en/controllers/components/pagination.rst [hunk 3]
```diff
@@ -234,19 +234,20 @@ additional details on how to use the table registry::
...
 
-    $published_articles = $this->paginate(
+    $publishedArticles = $this->paginate(
         $this->Articles->find('all', [
             'scope' => 'published_articles'
         ])->where('published' => true)
     );
     
-    $unplublished_articles = $this->paginate(
+    $unplublishedArticles = $this->paginate(
```
Comment (dereuromark) [inline en/controllers/components/pagination.rst]: plublish? :)

**Pull request merged**